CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Added the the `PassiveLocationManager.rawLocation` and `PassiveLocationManager.location` properties to get the latest raw and idealized locations, respectively. ([#3474](https://github.com/mapbox/mapbox-navigation-ios/pull/3474))
 * Fixed an issue where `ReplayLocationManager` would crash if initialized with just one location. ([#3528](https://github.com/mapbox/mapbox-navigation-ios/pull/3528))
 * Added the `ReplayLocationManager.onReplayLoopCompleted` property that allows you to stop the location manager from looping back to the beginning of the route. ([#3528](https://github.com/mapbox/mapbox-navigation-ios/pull/3528))
+* Fixed the route line flickering issue during active navigation when using default `.courseView` for `NavigationMapView.userLocationStyle`. ([#3533](https://github.com/mapbox/mapbox-navigation-ios/pull/3533))
```

Comment:
Is there an easy way to reproduce this issue?

Reply:
Not quite sure whether the https://github.com/mapbox/mapbox-navigation-ios/issues/3580 of the  course view drifting is related with it. But the route line flickering issue is observed by the customer during real navigation with the course view.